local-cli/runAndroid/runAndroid.js

```diff
@@ -271,6 +271,11 @@ function startServerInNewWindow() {
   const procConfig = {cwd: scriptsDir};
   const terminal = process.env.REACT_TERMINAL;
 
+  // setup the .packager.env file to ensure the packager starts on the right port
+  const packagerEnvFile = path.join(__dirname, '..', '..', 'scripts', '.packager.env');
+  const content = `export RCT_METRO_PORT=${port}`;
+  fs.writeFileSync(packagerEnvFile,content,{encoding: 'utf8', flag:'w'}); // passing the 'w' flag will overwrite the file
```

Comment:
Code style.

Reply:
This is my first PR for react-native, how would you like this styled differently?

Reply:
Maybe spaces in arguments on LOC 277 ?
ex: `packagerEnvFile,content` => `packagerEnvFile, content`

Reply:
@syzer fixed